testsuite/features/support/constants.rb

```diff
@@ -150,11 +150,11 @@
                            'sle15sp4_ssh_minion' => 'SLES15-SP4-Pool',
                            'sle15sp3_buildhost' => 'SLES15-SP3-Pool',
                            'sle15sp3_terminal' => 'SLES15-SP3-Pool',
-                           'ceos7_client' => 'RHEL x86_64 Server 7',
-                           'ceos7_minion' => 'RHEL x86_64 Server 7',
-                           'ceos7_ssh_minion' => 'RHEL x86_64 Server 7',
-                           'ceos8_minion' => 'no-appstream-result-RHEL8-Pool for x86_64',
-                           'ceos8_ssh_minion' => 'no-appstream-result-RHEL8-Pool for x86_64',
+                           'centos7_client' => 'RHEL x86_64 Server 7',
+                           'centos7_minion' => 'RHEL x86_64 Server 7',
+                           'centos7_ssh_minion' => 'RHEL x86_64 Server 7',
+                           'centos8_minion' => 'no-appstream-result-RHEL8-Pool for x86_64',
+                           'centos8_ssh_minion' => 'no-appstream-result-RHEL8-Pool for x86_64',
```

Comment:
:D You know what I think about this piece of code changes in that PR.

Reply:
Let's kill `ceos`, it's not readable. It's same issue ad `dblike` versus `deblike`.